src/docfx/Constants.cs

```diff
@@ -0,0 +1,12 @@
+namespace Microsoft.Docs.Build
+{
+    internal static class Constants
+    {
+        public static class Config
```

Comment:
Consider putting config constants inside config. we might not want to have a global `constants` class for everything.